packages/twenty-server/src/database/typeorm-seeds/workspace/connectedAccount.ts

```diff
@@ -0,0 +1,64 @@
+import { DataSource } from 'typeorm';
+
+const tableName = 'connectedAccount';
+
+export const seedConnectedAccount = async (
+  workspaceDataSource: DataSource,
+  schemaName: string,
+) => {
+  await workspaceDataSource
+    .createQueryBuilder()
+    .insert()
+    .into(`${schemaName}.${tableName}`, [
+      'id',
+      'createdAt',
+      'updatedAt',
+      'deletedAt',
+      'lastSyncHistoryId',
+      'accountOwnerId',
+      'refreshToken',
+      'accessToken',
+      'provider',
+      'handle',
+    ])
+    .orIgnore()
+    .values([
+    {
+      id: '6773281b-9ac0-4390-9a1a-ab4d2c4e1bb7',
+      createdAt: new Date(),
+      updatedAt: new Date(),
+      deletedAt: null,
+      lastSyncHistoryId: 'exampleLastSyncHistory',
+      accountOwnerId: '20202020-0687-4c41-b707-ed1bfca972a7',
```

Comment:
Instead of just copy/pasting ids between entities, you should do something like workspaceMemberIds:
```typescript
const WorkspaceMemberIds = {
  Tim: '20202020-0687-4c41-b707-ed1bfca972a7',
  Jony: '20202020-77d5-4cb6-b60a-f4a835a85d61',
  Phil: '20202020-1553-45c6-a028-5a9064cce07f',
};
```
This way you can reuse ids between entities and avoid misspelling issues